app/routes/verifier.py

```diff
@@ -484,19 +484,29 @@ async def delete_proof(
 )
 async def get_credentials_by_proof_id(
     proof_id: str,
+    referent: Optional[str] = None,
+    limit: Optional[str] = None,
+    offset: Optional[str] = None,
```

Comment:
Not sure if limit and offset are meant to be strings :-)

Also, we can define them as query params with validation: min value = 0 for both. max value = 10'000 for limit. something like that.

Then in unit tests, assert validation error is raised when passing bad limit/offset values

Reply:
Laat my weet wat jy dink van die toets wat ek ge-add het